### PR TITLE
[ES] Update `Client/Installation`

### PR DESCRIPTION
**Pull request by agatemosu**
The translation was not marked as outdated

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Installation/es.md [hunk 1]
```diff
@@ -1,81 +1,77 @@
...
+4. Localiza y abre el archivo del beatmap, tiene una extensión `.osz`.
+   - Alternativamente, también puedes colocar el archivo del beatmap en el directorio `Songs` de la instalación de osu!.
 5. osu! se encargará automáticamente del resto y cargará el beatmap.
-   - Si, por alguna razón, no los ves, presiona `F5` en [Modo Solo](/wiki/Client/Interface#song-select) para volver a procesar tus mapas de ritmo.
+   - Si por alguna razón no los ves, presiona `F5` en el [selector de canciones](/wiki/Client/Interface#selección-de-canciones) para volver a procesar tus beatmaps.
 
 ### osu!direct
 
-*Nota: Debes tener la etiqueta osu!supporter y la conexión a Internet para usar osu!direct.*
+*Nota: Debes tener una etiqueta osu!supporter y una conexión a Internet para usar osu!direct.*
```
Comment (UnknownOsu) [inline wiki/Client/Installation/es.md:53]:
```suggestion
*Nota: Debes tener una etiqueta de osu!supporter y una conexión a internet para usar osu!direct.*
```

File: wiki/Client/Installation/es.md [hunk 2]
```diff
@@ -1,81 +1,77 @@
...
-*Ver también: [Skins](/wiki/Skin) y [Skinning](/wiki/Skinning)*
+*Véase también: [Skinning](/wiki/Skinning)*
 
-1. Dirígete a [subforos skinning](https://osu.ppy.sh/community/forums/15) y busca el aspecto que más te guste.
-2. Una vez que encuentres un skin, descárgalo.
-3. Dependiendo de cómo el desollador haya empaquetado su skin, es posible que tenga que hacer cosas diferentes.
-   - Si la skin está usando la extensión `.osk`:
+1. Ve a los [foros de skinning](https://osu.ppy.sh/community/forums/15) y encuentra una skin que te guste.
+2. Una vez que encuentres una skin, descárgala.
+3. Dependiendo de cómo el desollador haya empaquetado su skin, es posible que debas hacer cosas diferentes.
```
Comment (UnknownOsu) [inline wiki/Client/Installation/es.md:68]:
```suggestion
3. Dependiendo de cómo el creador haya empaquetado su skin, es posible que debas hacer cosas diferentes.
```

File: wiki/Client/Installation/es.md [hunk 3]
```diff
@@ -1,81 +1,77 @@
...
+   - Si la skin está usando `.osk`:
      1. Ábrelo o arrástralo y suéltalo en osu!.
-     2. osu! procesará el skin y lo seleccionará automáticamente para ti.
-   - Si la skin está comprimida (usando las extensiones `.7z`,`.rar`, `.zip`, etc.):
-     1. Extraela.
-     2. Mueva la carpeta de la skin al directorio `Skins` de la instalacion de osu!.
-     3. Si osu! esta abierto, es posible que debas reiniciarlo antes de que se pueda usar la skin.
-     4. En osu!, ve a [Opciones](/wiki/Client/Options) y busca `Skin`.
+     2. osu! procesará la skin y lo seleccionará automáticamente para ti.
```
Comment (UnknownOsu) [inline wiki/Client/Installation/es.md:71]:
```suggestion
     2. osu! procesará la skin y la seleccionará automáticamente para ti.
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

File: wiki/Client/Installation/es.md (hunk 2 above)
Comment (UnknownOsu) [reply wiki/Client/Installation/es.md:68]: También puede ser que se deje en su forma original, a tu criterio.

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**